src/tbp/monty/frameworks/config_utils/make_dataset_configs.py

```diff
@@ -49,6 +59,7 @@ class ExperimentArgs:
     model_name_or_path: str = ""
     min_lms_match: int = 1
     seed: int = 42
+    supervised_lm_ids: Union[List[str], Literal["all"]] = field(default_factory=list)
```

Comment:
This looks fine to me. I forgot about setting defaults in a dataclass like this. I was thinking of the case of setting a default in a function argument list and the problems with using list or dictionary literals in those places. This is a bit different, and using the factory approach sidesteps the issue I was mentioning in our huddle.